.github/workflows/device-tests-android.yml

```diff
@@ -58,8 +58,9 @@ jobs:
       fail-fast: false
       matrix:
         tfm: [net9.0]
-        # We run against both an older and a newer API
-        api-level: [27, 33]
+        # Must be 34+ for new apps and app updates as of August 31, 2024.
+        # See https://apilevels.com/
+        api-level: [34, 36]
```

Comment:
question: simplify or keep (line: 95)

Do we still need the condition in line: 95
```YAML
target: ${{ matrix.api-level >= 32 && 'google_apis' || 'default' }}
```

or is it nice to keep `matrix.api-level >= 32`, although we're targeting `34` and `36`, so that the absence of `default` is documented

Reply:
True... I'll leave the comment but remove the conditional